jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+
+#### Is a Pipeline the Same Thing as a Build?
+
+In previous CircleCI release, the term "build" was used to describe a pipeline. Unfortunately, this terminology proved problematic because there was an incongruity between how "build" and "job" were defined in the API and UI, For example, what was referred to as a "build" in past releases is now referred to as a "job" in the current release. To address this ambiguity, CircleCI updated the UI to reflect this change in nomenclature, but there are still instances of "build" in our API.
+
+More importantly, CircleCI found that more teams are using CircleCI for a much broader array of automated processes than the word "build" implies. “Pipelines” more accurately reflects the broad applications of our platform, therefore, "pipelines" are seem more appropriate for the set of stages CircleCI performs in processing your automation instructions against changes in your code. 
+
+#### Will I Have to Use Pipelines?
+
+You are not required to use pipelines, but note that pipelines will eventually be enabled for all projects. This will allow CircleCI to provide more powerful configuration and automation, as well as better optimization of your resources and more reliable performance. The overarching goal of this approach is for the transition to pipelines to be seamless and have no disruptions on any of your existing workflows. 
+
+### Project Slugs
+
+#### What is a Project Slug?
+
+The CircleCI v2 API is backwards compatible with previous API versions in the way it identifies your projects using repository name. For example, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as `gh/CircleCI-Public/circleci-cli`, which is a "triplet" of the project type, the name of your "organization", and the name of the repository. 
+
+For the project type you can use `github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which are now supported in API v2. The `organization` is your username or organization name in your version control system.
+
+With API v2, CircleCI has introduced a string representation of the triplet called the `project_slug`, and takes the following form: 
+
+`<project_type>/<org_name>/<repo_name>`
+
+The `project_slug` is included in the payload when pulling information about a project as well as when looking up a pipeline or workflow by ID. The `project_slug` can then be used to get information about the project. It is possible in the future CircleCI may change the shape of a `project_slug`, but in all cases it would be usable as a human-readable identifier for a given project.
```

Comment:
I think this section is well explained. I think it would be cool to have a "pre 2.x" example curl call and a 2.x curl call so that users can contrast the new changes. Maybe when we get those code tabs implemented.

Reply:
There isn't really any change here from v1.1 to v2 API, really we're solidifying the notion of the project slug, it was implied before.